R/util.R

```diff
@@ -610,3 +610,49 @@ abbreviate <- function(x, len = round(getOption("width", 80) * 0.8)) {
   }
   x
 }
+
+handle_missing_packages <- function(missing_packages) {
+  ## check if we are interactive and logging is active...
+  logging <- !isTRUE(getOption("orderly.nolog"))
+  if (interactive() && logging) {
+    install_missing_packages(missing_packages)
```

Comment:
in order to make this testable, can you try adding an argument `force = FALSE` to `handle_missing_packages` then we can call `handle_missing_packages(packages, TRUE)` and trigger this line which remains untouched in tests